docs/source/installing-brave.rst

```diff
@@ -70,28 +70,28 @@ Beta Channel Installation
 
 .. highlight:: console
 
-Ubuntu 16.04+
--------------
+Ubuntu 16.04 and Mint 18+
```

Comment:
How about `Ubuntu 16.04+ and Mint 18+`? Its a little bit clearer, yours looks like its only Ubuntu 16.04 and Mint 18+

Reply:
Good feedback. Fixed in new revision.